tests/src/test/scala/DietSpec.scala

```diff
@@ -115,6 +115,13 @@ class DietTest extends FlatSpec with Matchers {
   import dogs.Predef._
   import Diet._
 
+  implicit object EnumInt extends Enum[Int] {
```

Comment:
we don't have to define this in the test, we have it in main:

https://github.com/stew/dogs/blob/master/core/src/main/scala/Enum.scala#L28